databricks/koalas/groupby.py

```diff
@@ -602,6 +602,79 @@ def rename_output(pdf):
             sdf=sdf, data_columns=return_schema.fieldNames(), index_map=[])  # index is lost.
         return DataFrame(internal)
 
+    def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
```

Comment:
@RainFung, seems this API should be under `SeriesGroupBy` since it's only supported in grouped series

Reply:
Yes.Is there some changes in doc rst file?